pkg/build/util/util.go

```diff
@@ -25,11 +27,11 @@ const (
 )
 
 // GetBuildName returns name of the build pod.
-func GetBuildName(pod *kapi.Pod) string {
+func GetBuildName(pod metav1.Object) string {
```

Comment:
though this *can* work on any object, there's an expectation that the caller is passing a pod.  I think it's better that the signature enforce that.  Otherwise someone could pass the wrong object and get a "valid" result back if that object happened to have the right annotation.